front-end/src/app/shared/models/transaction-types/FEDERAL_ELECTION_ACTIVITY_VOID.model.ts

```diff
@@ -12,7 +12,6 @@ export class FEDERAL_ELECTION_ACTIVITY_VOID extends SchBTransactionType {
   title = LabelUtils.get(ScheduleBTransactionTypeLabels, ScheduleBTransactionTypes.FEDERAL_ELECTION_ACTIVITY_VOID);
   schema = schema;
   override negativeAmountValueOnly = true;
-  override showAggregate = false;
```

Comment:
@toddlees had suggested adding `isRefundAggregate = true;` in the original CR which I think we should add now since we are showing the aggregate (it's n/a otherwise).

Reply:
@toddlees @dheitzer I am not convinced that this is the case. For refunds, this subtraction from the aggregate is explicitly stated as a requirement in the tickets. See here for an example: https://app.zenhub.com/workspaces/fecfile-online-619e578e68408b001c831251/issues/gh/fecgov/fecfile-web-app/723

I don't believe showing the aggregate on the form and subtracting the value of the transaction from the aggregate are tied so closely together. Maybe let's jump on a video about it.

Reply:
After looking at the ticket with @toddlees it was discovered that nether of these tickets is a refund so the isRefundAggregate flag should be false.